client/src/components/relatedProducts/relatedProductsList.jsx

```diff
@@ -22,13 +21,9 @@ const responsive = {
   }
 };
 
-
-
-//for new commit aug 30
-
 const RelatedProducts = (props) => {
   const [accumulatedProductData, setAccumulatedProductData] = React.useState([]);
-  const product_id = "37314"
+  const product_id = props.mainProduct
```

Comment:
again, consider deconstructing the props on line 24
`RelatedProducts = ({mainProduct})` then you can refer to it by name without the props.